flink-core/pom.xml

```diff
@@ -34,7 +34,14 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<surefire.module.config> <!--
+		<!--
+		Two properties below should be merged once support for jdk8 and jdk11 is dropped.
+		The reason of extraction into a separate property with profile is that
+		specifying -Djava.security.manager=allow within ${surefire.module.config} leads to
+		hanging of surefire for jdk8 and jdk11
```

Comment:
Would actually be interesting to know what is going on, because according to the JDK docs null/allow should be equivalent on JDK17-.

Reply:
It seems current surefire plugin (3.0.0-M5) swallows the error, i bumped locally to 3.0.0-M6 and received a bit more info 
```
Error occurred during initialization of VM
java.lang.Error: Could not create SecurityManager
	at java.lang.System.initPhase3(java.base@11.0.20.1/System.java:2078)
Caused by: java.lang.ClassNotFoundException: allow
	at jdk.internal.loader.BuiltinClassLoader.loadClass(java.base@11.0.20.1/BuiltinClassLoader.java:581)
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(java.base@11.0.20.1/ClassLoaders.java:178)
	at java.lang.ClassLoader.loadClass(java.base@11.0.20.1/ClassLoader.java:527)
	at java.lang.Class.forName0(java.base@11.0.20.1/Native Method)
	at java.lang.Class.forName(java.base@11.0.20.1/Class.java:398)
	at java.lang.System.initPhase3(java.base@11.0.20.1/System.java:2063)
```

Reply:
Well that's just stupid. 😮‍💨